File: app/src/main/java/com/babestudios/companieshouse/data/model/insolvency/Address.java
package com.babestudios.companieshouse.data.model.insolvency;


import com.google.gson.annotations.SerializedName;

public class Address {
	@SerializedName("address_line_1")
	public String addressLine1;
	@SerializedName("address_line_2")
	public String addressLine2;
	@SerializedName("country")
	public String country;
	@SerializedName("locality")
	public String locality;
	@SerializedName("postal_code")
	public String postalCode;
	@SerializedName("region")
	public String region;
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/insolvency/Address.kt
package com.babestudios.companyinfouk.data.model.insolvency


import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Address(
		@SerializedName("address_line_1")
		var addressLine1: String? = null,
		@SerializedName("address_line_2")
		var addressLine2: String? = null,
		@SerializedName("country")
		var country: String? = null,
		@SerializedName("locality")
		var locality: String? = null,
		@SerializedName("postal_code")
		var postalCode: String? = null,
		@SerializedName("region")
		var region: String? = null
) : Parcelable
